### DISCUSSION TITLE
"How I had a nerdy date night with StreetComplete quests" article

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2558] [General]
https://opensource.com/article/20/1/streetcomplete-crowdsource-maps

> If you live in a mobile deadzone or have a limited mobile plan, you can still have fun questing for map data; StreetComplete has an offline mode. To use it, activate the offline download option in the app settings menu. Once your region has downloaded, perform a search for nearby quests, then go out and explore.

Seems to indicate something a bit confusing in interface.

**Comment by westnordost** [comment]
The article is over one year old
